ui/src/components/policy/AddAssertion.js

```diff
@@ -87,7 +88,7 @@ export default class AddAssertion extends React.Component {
             )
             .then((data) => {
                 this.props.submit(
-                    `${this.props.name}-${this.state.role}-${this.state.resource}-${this.state.action}`,
+                    `${this.props.name}-${this.state.version}-${this.state.role}-${this.state.resource}-${this.state.action}`,
```

Comment:
this.props.version instead of state.version

Reply:
fixed